test/index.js

```diff
@@ -741,6 +741,42 @@ it('accepts request object in validateFunc', (done) => {
     });
 });
 
+it('includes additional attributes in WWW-Authenticate header', (done) => {
+    const server = new Hapi.Server();
```

Comment:
Local issue :flushed:  `rm -r node_modules/ && npm i && npm test` caught it like it should have. found that `hapi-scope-start` has [a nice test](https://github.com/continuationlabs/hapi-scope-start/blob/master/test/index.js#L200) for exactly this